src/sidebar/components/test/group-list-test.js

```diff
@@ -454,5 +432,56 @@ describe('groupList', function() {
       assert.lengthOf(dropdownMenu, 1);
       assert.lengthOf(dropdownOptions, 2);
     });
+
+    it('is shown when community_groups feature flag is on and there are multiple groups', function() {
+      fakeFeatures.flagEnabled.withArgs('community_groups').returns(true);
+      groups = [publicGroup, restrictedGroup];
+
+      const element = createGroupList();
+
+      const showGroupsMenu = element.ctrl.showGroupsMenu();
+      const dropdownToggle = element.find('.dropdown-toggle');
+      const arrowIcon = element.find('.h-icon-arrow-drop-down');
+      const dropdownMenu = element.find('.dropdown-menu__top-arrow');
+      const dropdownOptions = element.find('.dropdown-menu__row ');
+
+      assert.isTrue(showGroupsMenu);
+      assert.lengthOf(dropdownToggle, 1);
+      assert.lengthOf(arrowIcon, 1);
+      assert.lengthOf(dropdownMenu, 1);
+      assert.lengthOf(dropdownOptions, 3);
+    });
+
+    it('is not shown when community_groups feature flag is on and there is only one group', function() {
+      fakeFeatures.flagEnabled.withArgs('community_groups').returns(true);
+      groups = [privateGroup];
+
+      const element = createGroupList();
+
+      const showGroupsMenu = element.ctrl.showGroupsMenu();
+      const dropdownToggle = element.find('.dropdown-toggle');
+      const arrowIcon = element.find('.h-icon-arrow-drop-down');
+      const dropdownMenu = element.find('.dropdown-menu__top-arrow');
+      const dropdownOptions = element.find('.dropdown-menu__row ');
+
+      assert.isFalse(showGroupsMenu);
+      assert.lengthOf(dropdownToggle, 0);
+      assert.lengthOf(arrowIcon, 0);
+      assert.lengthOf(dropdownMenu, 0);
+      assert.lengthOf(dropdownOptions, 0);
+    });
+  });
+
+  [false, true].forEach(isEnabled => {
```

Comment:
Given that `flagEnabled` is a stub, I'm not clear on what this test tests (in a meaningful sense)...would it be more accurate to describe it as "it returns what flagEnabled returns"?...

Reply:
That's true, it's basically just making sure we've wrappered the features.flagEnabled correctly.